src/System.IO.Ports/src/System/IO/Ports/SerialStream.cs

```diff
@@ -805,13 +800,15 @@ protected override void Dispose(bool disposing)
                         {
                             _handle.Close();
                             _handle = null;
+                            _threadPoolBinding.Dispose();
```

Comment:
Per offline conversation: it could be important in a finalizer and double disposal scenario but nothing that demands to be changed here, will re-evaluate when porting some of the changes here back to master.